internal/web/commands.go

```diff
@@ -36,26 +36,8 @@ func saveOCI(dst, image string) string {
 }
 
 func runBashProcessWithOutput(ws io.Writer, command string) error {
-	// Simulate a background process
 	cmd := exec.Command("bash", "-c", command)
-	stdout, _ := cmd.StdoutPipe()
-	stderr, _ := cmd.StderrPipe()
-
-	out := io.MultiReader(stdout, stderr)
-
-	if err := cmd.Start(); err != nil {
-		return err
-	}
-
-	// Stream process output to writer
-	reader := io.TeeReader(ansiToHTML(out), ws)
```

Comment:
I think the problem was caused by overflowing the TeeReader but I'm not sure how to fix that, so in the end i decided to just send all output to the channel and do the parsing of ansi to html in the frontend with a library which seems maintained properly